docs/api/resnet_pretrained_weights.csv

```diff
@@ -2,9 +2,29 @@ Weight,Channels,Source,Citation,BigEarthNet,EuroSAT,So2Sat,OSCD
 ResNet18_Weights.SENTINEL2_ALL_MOCO,13,`link <https://github.com/zhu-xlab/SSL4EO-S12>`__,`link <https://arxiv.org/abs/2211.07044>`__,,,,
 ResNet18_Weights.SENTINEL2_RGB_MOCO, 3,`link <https://github.com/zhu-xlab/SSL4EO-S12>`__,`link <https://arxiv.org/abs/2211.07044>`__,,,,
 ResNet18_Weights.SENTINEL2_RGB_SECO, 3,`link <https://github.com/ServiceNow/seasonal-contrast>`__,`link <https://arxiv.org/abs/2103.16607>`__,87.27,93.14,,46.94
+ResNet18_Weights.LANDSAT_ETM_SR_MOCO,6,`link <https://github.com/microsoft/torchgeo>`__,`link <https://arxiv.org/abs/2306.09424>`__,,,,
```

Comment:
It doesn't really make sense to add all ResNet metrics to a single table. We can't report model performance of Landsat weights on BigEarthNet/EuroSAT/So2Sat/OSCD because those are completely different datasets. I think we want a separate table for each sensor instead of each model. In this case, that would mean one table for Sentinel 1, one for Sentinel 2, and one for all of Landsat. We can't combine S1 and S2 because the datasets used to evaluate them are different. But we can combine L5–L9 because we have a single SSL4EO-L CDL/NLCD dataset that we can directly compare across.

Reply:
what do you want to do with ResNet50_Weights.FMOW_RGB_GASSL?

Reply:
I forget, did we decide it was trained on Planet, or on many different satellites? If the latter, we could choose one of the following options:

* Single table for all miscellaneous weights
* Single table for all high-res RGB weights
* Separate table for all miscellaneous datasets

Since it's only one entry right now, I don't particularly care which we use. We can change it later.